tardis/io/util.py

```diff
@@ -209,32 +209,44 @@ def to_hdf_util(path_or_buf, path, elements, complevel=9, complib="blosc"):
 
         Parameters
         ----------
-        path_or_buf :
-            Path or buffer to the HDF store
+        path_or_buf : str or pandas.io.pytables.HDFStore
+            Path or buffer to the HDF file
         path : str
-            Path inside the HDF store to store the `elements`
+            Path inside the HDF file to store the `elements`
         elements : dict
             A dict of property names and their values to be
             stored.
-
-        Returns
-        -------
+        overwrite: bool
+            If the HDF file path already exists, whether overwrite it or not
+
+        Notes
+        -----
+        `overwrite` option doesn't have any effect when `path_or_buf` is an
+        HDFStore because it depends on user that in which mode they've opened
+        it.
```

Comment:
`overwrite` option doesn't have an effect when `path_or_buf` is an HDFStore because the user decides on the mode in which they have opened the HDF (path or HDFStore).

Reply:
Umm...actually I meant, file opening mode ('r', 'w', 'a') - when specifying HDFStore user has control over the mode in their hands, so `overwrite` option won't have any effect.

Reply:
Rephrased it - let me know if it still needs changes!